TweetBook/Contracts/V1/ApiRoutes.cs

```diff
@@ -9,12 +9,15 @@ public static class ApiRoutes
         public const string Version = "v1";
 
         public const string Base = $"{Root}/{Version}";
+        // public const string Base = Root + "/" + Version;
 
         public static class Posts
         {
             public const string GetAll = $"{Base}/posts";
-            public const string Create = "api/v1/posts";
-            public const string Get = "api/v1/posts/{postId}";
+            public const string Get = Base + "/posts/{postId}";
+            public const string Create = Base + "/posts";
+            public const string Update = Base + "/posts/{postId}";
+            public const string Delete = Base + "/posts/{postId}";

```

Comment:
Try to be consistent, use only one type of concatenation, preferrably $"{}"

Reply:
Please, is for me, is my style of learning, I understand that in a real project is not the way of doing.